editor/src/clj/editor/app_view.clj

```diff
@@ -1870,6 +1870,22 @@ If you do not specifically require different script states, consider changing th
 (defn- tab->view-type [^Tab tab]
   (some-> tab (ui/user-data ::view-type) :id))
 
+(defn- get-target-tab
+  [^MouseEvent event]
+  (let [TabHeaderSkin (Class/forName "javafx.scene.control.skin.TabPaneSkin$TabHeaderSkin")
+        getTab (.getDeclaredMethod TabHeaderSkin "getTab" (into-array Class []))]
+    (.setAccessible getTab true)
```

Comment:
I think we should not do class/method reflection on every call to `get-target-tab`. Instead, we should make it a top-level statement, like here: https://github.com/defold/defold/blob/4e8483c3143df92f66ae6086d4d2cfb0e843c8a7/editor/src/clj/editor/lsp.clj#L599-L604
This way, it's both more performant in the runtime, and if a dependency update breaks our use of reflection, we will know about it immediately when the app is started.